test/fixtures/output/flink/compute-pool/describe-after-use.golden

```diff
@@ -4,6 +4,7 @@
 | Name        | my-compute-pool-1 |
 | Current CFU | 0                 |
 | Max CFU     | 1                 |
+| Cloud       | AWS               |
```

Comment:
Why is "AWS" capitalized here but not in the output for `create`? Hopefully the backend standardizes this?

Reply:
Yeah backend returns everything in uppercase, I believe the reason why the create case is lowercase is because the tests are generated from this command:
`flink compute-pool create my-compute-pool --cloud aws --region us-west-2` 
And I think the fake backend just uses whatever the input CP was and then returns it 🤔

Reply:
Do you mind uppercasing it so it's more realistic?

Reply:
Should be fixed now!